src/App.css

```diff
@@ -1,4 +1,126 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import 'react-circular-progressbar/dist/styles.css';
+
 * {
+  box-sizing: border-box;
   margin: 0;
   padding: 0;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  padding: 1.5rem;
+  background-color: #fafafa;
+}
+
+.App {
+  width: 87.5rem;
+  height: 68.563rem;
+  margin: 0 2.5rem 4.063rem 0;
+  padding: 0 0 49.625rem;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 var(--black-5);
+}
+
+.navBar-container {
+  background-color: #fff;
+  border-bottom: #fafafa 1px solid;
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  color: #0290ff;
+  font-size: 1.875rem;
+  font-weight: bold;
+}
+
+.navLinks {
+  display: flex;
+  gap: 2.563rem;
+  margin-right: 620px;
+  align-items: center;
+}
+
+.link {
+  list-style: none;
+}
+
+.link a {
+  text-decoration: none;
+  color: #848484;
+}
+
+.link a:active {
+  color: #000;
+}
+
+.books-container li {
+  list-style: none;
+}
+
+.bigContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 75rem;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.title {
+  margin: 0.188rem 20.188rem 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.author {
+  margin: 0.35rem 0 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.comment,
+.remove,
+.edit {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  margin: 20px 20px 0;
+  border: 0;
+  background-color: #fff;
+}
+
+.progressBar {
+  width: 4.25rem;
+  height: 4.25rem;
+}
+
+.progressContainer {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
+
+.percentage {
+  font-size: 2rem;
+}
+
+.comp {
+  opacity: 0.5;
+  font-size: 0.875rem;
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you stick to the zeplin template. Let's make sure that the Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)
Take a look at the button and it's margin..Please ensure you use the same color, font-family , font-size, font-weight and all other properties as stated on Zeplin.
### Zeplin
<img width="1035" alt="Screenshot 2022-11-26 at 23 56 57" src="https://user-images.githubusercontent.com/58692590/204111899-03fec9fa-0a1d-433c-a1ca-9ec0e5a8c902.png">
### Your design
<img width="1262" alt="Screenshot 2022-11-27 at 00 01 11" src="https://user-images.githubusercontent.com/58692590/204111917-e815a8e6-cee5-4990-bc92-e6c7b0597f89.png">